sdk/ai/azure-ai-resources/azure/ai/resources/github/accesstoken/get_github_token.py

```diff
@@ -0,0 +1,64 @@
+# ---------------------------------------------------------
```

Comment:
As we discussed offline. Please move both files to this folder in ML SDK. https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/ml/azure-ai-ml/azure/ai/ml/_utils

Reply:
moved